dataprofiler/tests/profilers/profiler_options/test_hyper_log_log_options.py

```diff
@@ -0,0 +1,185 @@
+from dataprofiler.profilers.profiler_options import HyperLogLogOptions
+from dataprofiler.tests.profilers.profiler_options.test_boolean_option import (
+    TestBooleanOption,
+)
+
+
+class TestHyperLogLogOptions(TestBooleanOption):
+
+    option_class = HyperLogLogOptions
+
+    def test_init(self):
+        option = self.get_options()
+        self.assertDictEqual(
+            {"is_enabled": False, "seed": 0, "register_count": 10}, option.properties
+        )
+        option = self.get_options(is_enabled=True)
+        self.assertDictEqual(
+            {"is_enabled": True, "seed": 0, "register_count": 10}, option.properties
+        )
+        option = self.get_options(seed=10, register_count=9)
+        self.assertDictEqual(
+            {"is_enabled": False, "seed": 10, "register_count": 9}, option.properties
+        )
+
+    def test_set_helper(self):
+        super().test_set_helper()
+
+    def test_set(self):
+        super().test_set()
+        option = self.get_options()
+        option.set({"is_enabled": True, "seed": 5, "register_count": 8})
+        self.assertDictEqual(
+            {"is_enabled": True, "seed": 5, "register_count": 8}, option.properties
+        )
+
+    def test_validate_helper(self):
+        super().test_validate_helper()
+
+        optpth = self.get_options_path()
+
+        # Default configuration
+        option = self.get_options(seed=0, register_count=10)
```

Comment:
In order to test default, I suggest not overriding. i.e. just do `option = self.get_options()`

Reply:
+1

Reply:
fixed